### PR TITLE
OCPBUGS-1507 making updates for --offlined-cpu-count

### PR DESCRIPTION
**Pull request by kquinn1204**
OCPBUGS-1507: PPC documentation missing offlined parameter

Version(s):
  * PR applies only to a specific single version: 4.11

Issue: 

- https://issues.redhat.com/browse/OCPBUGS-1507

Link to docs preview: 

- http://file.emea.redhat.com/kquinn/OCPBUGS-1507/scalability_and_performance/cnf-create-performance-profiles.html
- https://50795--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-create-performance-profiles.html

NOTES: Dev and QE review complete

### CODE REVIEW

**Comment by kquinn1204** [comment]
@mrniranjan It seems we missed the --offline parameter from PPC tool. I added it back in. Can you please review?

**Comment by kquinn1204** [comment]
@marioferh Can you please review this looks like we missed the update of `--offline-cpu-count` in PPC section of docs

**Comment by mrniranjan** [comment]
LGTM

File: modules/cnf-performance-profile-creator-arguments.adoc [hunk 1]
```diff
@@ -47,6 +47,14 @@ Default: `log`.
 
 When the user runs the tool with the wrapper script `must-gather` is supplied by the script itself and the user must not specify it.
 
+| `offlined-cpu-count`
+a| Number of offlined CPUs.
```
Comment (marioferh) [inline modules/cnf-performance-profile-creator-arguments.adoc:51]: maybe it worth to explain who is calculated?
To calculate offlined cpus we take into account the following input parameters:

    offlined-cpu-count : the number of cpus the user wants to set offline.
    disable-ht: Marks if the user want to disable logical processor siblings to disable hyperthreading
    power-consumption-mode: values: “default”, “low-latency”, “ultra-low-latency”

https://github.com/openshift/cluster-node-tuning-operator/pull/354#issuecomment-1129124730

Comment (kquinn1204) [reply modules/cnf-performance-profile-creator-arguments.adoc:51]: Hi @marioferh I found it difficult to get that into the docs without getting into the weeds I hope that is OK. I did note the error messages that will be printed.

Comment (marioferh) [reply modules/cnf-performance-profile-creator-arguments.adoc:51]: yep sure, the docs are well explained. It was just to go a little further.

**Review by marioferh — COMMENTED** [review]
/lgtm

**Review by kelbrown20 — COMMENTED** [review]
Looks great! 
/lgtm

**Comment by bergerhoffer** [comment]
@stevsmit @kquinn1204 It looks like this was merged but not cherry picked. Does it still need to be?

Also, I see that this was marked for 4.11 only, but if it went into main, then it needs to go into 4.12 too. If it really was for 4.11 only, then it should have gone directly into enterprise-4.11.

**Comment by kquinn1204** [comment]
@bergerhoffer @stevsmit Sorry my bad I was so focussed on ensuring it did not go into any release before 4.11 I forgot to note it should go in releases after 4.11 (main and 4.12). The feature was only introduced in 4.11 into PAO.

**Comment by bergerhoffer** [comment]
Thanks @kquinn1204! I'll cherry pick to 4.11 and 4.12 now then

**Comment by bergerhoffer** [comment]
@kquinn1204 cherry picks are giving us trouble right now. I'm going to keep this window open and try again in a little bit, but if you can please also check back on this PR by tomorrow and follow up with me again if you see it doesn't get cherry picked, I'd appreciate it!

**Pull request merged**